platform/core/src/services/MeasurementService/MeasurementService.js

```diff
@@ -0,0 +1,125 @@
+import log from '../../log';
+import guid from '../../utils/guid';
+
+const EVENTS = {
+  MEASUREMENT_UPDATED: 'event@measurement_updated',
+  MEASUREMENT_ADDED: 'event@measurement_added',
```

Comment:
Bike shedding a little, but `event::measurement_added` may be slightly easier to read; if we're using a delimiter for "namespacing" / hierarchy